packages/synapse-react-client/src/lib/containers/oauth/CreateOAuthClient.tsx

```diff
@@ -199,246 +213,237 @@ export const CreateOAuthModal: React.FunctionComponent<
-                      </Form.Label>
-                      <Tooltip title={uriHelpMessage} placement="top">
-                        <HelpOutlineTwoTone className={`HelpButton`} />
-                      </Tooltip>
+  const sectorIdentifierURIInput = (
+    <StyledFormControl fullWidth variant="standard" margin="normal">
+      <TextField
+        onChange={e => setSectorUri(e.target.value)}
+        placeholder="https://"
+        type="text"
+        value={sectorUri}
+        id="sectorURI"
+        label="Sector Identifier URI"
+        InputLabelProps={{ sx: dangerLabelSx }}
+      />
+    </StyledFormControl>
+  )
 
-                      {redirectUris?.map((singleUri, idx) => (
-                        <div key={idx}>
-                          <Form.Control
-                            id={`redirect-uri-${idx}`}
-                            required
-                            name="uri"
-                            onChange={e => handleUriChange(e, idx)}
-                            value={singleUri.uri}
-                            placeholder="https://"
-                            type="text"
-                          />
-                          {redirectUris.length > 1 && (
-                            <button
-                              onClick={() => handleRedirectUriRemove(idx)}
-                            >
-                              <IconSvg
-                                icon="delete"
-                                sx={{ color: 'error.main' }}
-                              />
-                            </button>
-                          )}
+  const redirectURIInputs = (
+    <StyledFormControl fullWidth variant="standard" margin="normal">
+      {redirectUris?.map((singleUri, idx) => (
+        <div key={idx}>
+          <TextField
+            required={idx === 0}
+            InputLabelProps={{
+              sx: { ...dangerLabelSx, pointerEvents: 'auto', zIndex: 1 },
+            }}
+            label={
+              idx === 0 && (
+                <>
+                  Redirect URI(s)
+                  <Tooltip title={uriHelpMessage} placement="top">
+                    <HelpOutlineTwoTone
+                      className={`HelpButton`}
+                      sx={{ float: 'right' }}
+                    />
+                  </Tooltip>
+                </>
+              )
+            }
+            name="uri"
+            fullWidth
+            id={`redirect-uri-${idx}`}
+            onChange={e => handleUriChange(e, idx)}
+            value={singleUri.uri}
+            placeholder="https://"
+            type="text"
+            InputProps={{
+              endAdornment: redirectUris.length > 1 && (
+                <InputAdornment position="end">
+                  <IconButton onClick={() => handleRedirectUriRemove(idx)}>
+                    <DeleteTwoToneIcon sx={{ color: 'error.main' }} />
+                  </IconButton>
+                </InputAdornment>
+              ),
+            }}
```

Comment:
Use endAdornment for delete icon, rather than positioning next to the input field